### PR TITLE
Enhance Planned Enhancement

### PR DESCRIPTION
**Pull request by TechRaj**
update enhancement 1 to include accented characters as well

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
 
 Team size: 5
 
-1\. **Allow names to include symbols and special characters**
+1\. **Enhance Name Validation to Support Special Characters and Improve Formatting**
```
Comment (PresleyChew) [inline docs/DeveloperGuide.md:1118]: remove caps from all words in "Support Special characters and Improve Formatting"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
...
 
-  As of the current version of MediBase3, names are restricted to both alphanumerics and spaces only. 
-  However, there are people with legal names that include symbols and special characters, such as 'Nagaratnam s/o Suppiah'. In the future, we will loosen the restrictions for names to allow users to input names with any special characters and symbols.
+   Currently, MediBase3 restricts patient names to alphanumerics and spaces only, preventing the inclusion of common symbols, hyphens, and accented characters that are often found in legal names (e.g., 'Nagaratnam s/o Suppiah', 'Anya Taylor-Joy', 'Sergio Pérez'). Additionally, the app allows leading, trailing, and multiple consecutive spaces, which can result in inconsistent formatting.
```
Comment (PresleyChew) [inline docs/DeveloperGuide.md:1120]: lol sergio perez

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1115,10 +1115,15 @@ Deleting an existing appointment from a patient
...
+
+   This enhancement will:
+    - Loosen restrictions to allow symbols, hyphens, and accented characters, enabling the accurate entry of a wider range of legitimate names.
```
Comment (yong-zhuo) [inline docs/DeveloperGuide.md:1123]: Maybe can specify 'all other special characters'

**Review by PresleyChew — COMMENTED** [review]
Overall LGTM, just some nits on capital letters

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (yong-zhuo) [inline docs/DeveloperGuide.md:1118]: Could possible just rephrase this to 'Enhance Name Validation' so that it is not interpreted as 2 separate enhancements

Comment (PresleyChew) [reply docs/DeveloperGuide.md:1118]: self amend

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (PresleyChew) [reply docs/DeveloperGuide.md]: done

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (PresleyChew) [reply docs/DeveloperGuide.md]: Amended

**Review by yong-zhuo — APPROVED** [review]

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (TechRaj) [reply docs/DeveloperGuide.md]: for which para?

**Review by yadobler — APPROVED** [review]
lgtm
